apis/v1beta1/config.go

```diff
@@ -238,6 +251,21 @@ func (c *Config) Yaml() (string, error) {
 	return buf.String(), nil
 }
 
+func (c Config) MarshalJSON() ([]byte, error) {
+	type Alias Config
+	receiversJSON, err := json.Marshal(c.Receivers)
+	if err != nil {
+		return nil, err
+	}
+	return json.Marshal(&struct {
+		*Alias
+		Receivers json.RawMessage `json:"receivers,omitempty"`
+	}{
+		Alias:     (*Alias)(&c),
+		Receivers: receiversJSON,
+	})
+}
```

Comment:
Why are receivers singled out here? Shouldn't this issue potentially apply to all component kinds?